crates/ruff_server/src/trace.rs

```diff
@@ -82,15 +121,25 @@ pub(crate) fn init_tracing(
                 .ok()
         });
 
+    let logger = match log_file {
+        Some(file) => BoxMakeWriter::new(Arc::new(file)),
+        None => {
+            if client
+                .as_ref()
+                .is_some_and(|client| client.name.starts_with("Zed ") || client.name == "Zed")
```

Comment:
nit: can we combine both `starts_with` and `==` into a single check that just uses `starts_with("Zed")`? I think that should work as per the possible display names:

https://github.com/zed-industries/zed/blob/b7c6f3e98eb53eb516cba9bb1724ebbbf780cfd4/crates/release_channel/src/lib.rs#L129-L137

Reply:
And, I think we should also include "Visual Studio Code" here because it also supports `$/logTrace` requests.

Reply:
Very interesting that there's no `ClientCapability` indicating whether `logTrace` is supported.